### PR TITLE
[1749] Add Allocations documentation

### PR DESCRIPTION
**Pull request by AbigailMcP**
### Context

https://trello.com/c/Lfmb9xtS/1749-document-allocations-steps

### Changes proposed in this pull request

- Adds documentation for Allocations

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [ ] Tested by running locally **N/A**
- [ ] Product Review **N/A**

### CODE REVIEW

**Comment by AbigailMcP** [comment]
[Document Allocations steps](https://trello.com/c/Lfmb9xtS/1749-document-allocations-steps)

File: docs/allocations.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Allocations
+
+Each year, around the same time as [Rollover](./rollover.md), providers request
```
Comment (defong) [inline docs/allocations.md:3]:
```suggestion
Each academic year, around the same time as [Rollover](./rollover.md), providers request
```

I think we need make a slight distinction with what is that `year`.

File: docs/allocations.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+places for courses that have numbers of applicants limited as they are generally
+over-supplied. This process is called 'Allocations'.
+
+This document lists the changes needed to be made to the Publish codebase and
+the timings for these changes.
+
+## Before Allocations
+
+- Update any hardcoded dates relating to Allocation years and timelines
```
Comment (defong) [inline docs/allocations.md:12]: I think we need to make some references to academic year

Comment (AbigailMcP) [reply docs/allocations.md]: done!

File: docs/allocations.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Allocations
+
+Each year, around the same time as [Rollover](./rollover.md), providers request
+places for courses that have numbers of applicants limited as they are generally
+over-supplied. This process is called 'Allocations'.
```
Comment (defong) [inline docs/allocations.md:5]:
```suggestion
Each year, around the same time as [Rollover](./rollover.md), providers request
places for courses. This is to limited applicants as they are generally
over-supplied for that particular type of courses. This process is called 'Allocations'.
```

File: docs/allocations.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+This document lists the changes needed to be made to the Publish codebase and
+the timings for these changes.
+
+## Before Allocations
+
+- Update any hardcoded dates relating to the academic year the Allocations are
+  for e.g. Allocations made in the year 2021 will be for the academic year
+  2022 to 2023. [TBC - this may be refactored to be dynamic based on a setting]
```
Comment (defong) [inline docs/allocations.md:14]: @mooktakim :eyes:

Comment (AbigailMcP) [reply docs/allocations.md]: I've updated!

**Review by defong — APPROVED** [review]
LGTM

**Pull request merged**